doc/news/changes/minor/20190808KatherineCosburn

```diff
@@ -0,0 +1,7 @@
+The original documentation in step-2 under "possible extensions" states in the second paragraph: "Or, you could see how the sparsity pattern changes with more refinements." 
+
+I found this to be redundant and confusing given that, in the paragraph before it, we are asked to explore how the sparsity pattern changes when we use higher order elements in the distribute_dofs function (ie. p-refinement).
+
+I assumed that what was meant in this second paragraph was to explore how the pattern changes when we refine the mesh (ie. h-refinement) so I edited to make this more clear.
+
+(Katherine Cosburn, 2019/08/08)
```

Comment:
Could you try to be a little bit more concise in the changelog entry? We normally try to describe changes in one or two sentences.

Reply:
Yes! Sorry, I hope my newest version suitably concise.